### DISCUSSION TITLE
How to use domain from godaddy in vercel

### DISCUSSION THREAD

**Discussion by kumaresan-subramani** [vercel/vercel#6013] [Help] [answered]
i bought domain in godaddy, i want to use that here.

I need  proper steps, is ther any docs/video available for this

**Comment by mcsdevv** [comment, marked as answer]
Hi @kumaresan-subramani, please see the documentation on this: https://vercel.com/docs/projects/domains/add-a-domain

**Comment by thefonso** [comment]
i bought domain in godaddy, i want to use that here.

I need proper steps, is ther any docs/video available for this

Same question...

**Reply by mcsdevv** [reply]
Hey @thefonso, the answer in the thread is still the best advice here - are you running into any difficulty with it?
